collect_app/src/main/java/org/odk/collect/android/widgets/ItemsetWidget.java

```diff
@@ -234,46 +238,68 @@ protected ItemsetWidget(Context context, FormEntryPrompt prompt, boolean readOnl
                         mAnswers.put(label, val);
 
                         RadioButton rb = new RadioButton(context);
+
                         rb.setOnCheckedChangeListener(this);
-                        rb.setText(label);
                         rb.setTextSize(mAnswerFontsize);
-                        mButtons.addView(rb);
+                        rb.setText(label);
+                        rb.setTag(index);
+                        rb.setId(QuestionWidget.newUniqueId());
+
+                        mButtons.add(rb);
+
                         // have to add it to the radiogroup before checking it,
                         // else it lets two buttons be checked...
                         if (currentAnswer != null
                                 && val.compareTo(currentAnswer) == 0) {
                             rb.setChecked(true);
                         }
+
+                        MediaLayout mediaLayout = new MediaLayout(getContext(), mPlayer);
```

Comment:
Can this just be a `RelativeLayout` since there is no possible media for now? Even if this must use a `MediaLayout` for some reason, it doesn't make sense to set the audio listener, play text color, etc.